tests/fake_messages/pipeline_failure_message.json

```diff
@@ -21,7 +21,6 @@
   "body": {
     "msg": {
       "category": "functional",
-      "status": "FAILED",
```

Comment:
Why was this line removed?

Reply:
The is no "status" attribute for these "error" messages (see [Factory 2.0 CI UMB messages](https://docs.google.com/document/d/16L5odC-B4L6iwb9dp8Ry0Xk5Sc49h9KvTHrG86fdfQM/edit#heading=h.gjun9319pfoj)).